hatch/domain.py

```diff
@@ -0,0 +1,26 @@
+class Domain(object):
```

Comment:
I think now is a good time to add some py.test unit tests... Example: does this work for a .co.uk domain?